### PR TITLE
Removed duplicate structs, changed AudioRendererResponse -> UpdateDataHeader

### PR DESCRIPTION
**Pull request by ogniK5377**
According to game symbols(SMO), there's references to UpdateDataHeader which seems to be what AudioRendererResponse actually is

### CODE REVIEW

File: src/core/hle/service/audio/audren_u.cpp [hunk 1]
```diff
@@ -58,22 +58,22 @@ class IAudioRenderer final : public ServiceFramework<IAudioRenderer> {
     }
 
     void RequestUpdateAudioRenderer(Kernel::HLERequestContext& ctx) {
-        AudioRendererConfig config;
+        UpdateDataHeader config{};
```
Comment (mailwl) [inline src/core/hle/service/audio/audren_u.cpp]: why need this mess. Defintion on RequestUpdateAudioRenderer, from, for example, Axiom Verge:
`.text:000000710017E24C ; nn::audio::RequestUpdateAudioRenderer(nn::audio::AudioRendererHandle, nn::audio::AudioRendererConfig const*)`

Comment (ogniK5377) [reply src/core/hle/service/audio/audren_u.cpp]: AudioRendererConfig seems to hold the input and the output buffer as well as some other pointers. It doesn't seem like the correct struct name

Comment (ogniK5377) [reply src/core/hle/service/audio/audren_u.cpp]: `.text:000000710018E140 ; nn::audio::server::InfoUpdater::CheckConsumedSize(void)	
.text:000000710018D250 ; nn::audio::server::InfoUpdater::InfoUpdater(ulong,ulong,ulong,ulong,uint,nn::audio::server::BehaviorInfo &)	
.text:000000710018E05C ; nn::audio::server::InfoUpdater::UpdateBehaviorInfo(nn::audio::server::BehaviorInfo *)	
.text:000000710018D7BC ; nn::audio::server::InfoUpdater::UpdateEffects(nn::audio::server::EffectContext &,bool,nn::audio::server::MemoryPoolInfo *,int)	
.text:000000710018E0E8 ; nn::audio::server::InfoUpdater::UpdateErrorInfo(nn::audio::server::BehaviorInfo *)	
.text:000000710018DE98 ; nn::audio::server::InfoUpdater::UpdateMemoryPools(nn::audio::server::MemoryPoolInfo *,int)	
.text:000000710018DB3C ; nn::audio::server::InfoUpdater::UpdateMixes(nn::audio::server::MixContext &,int,nn::audio::server::EffectContext const&,nn::audio::server::SplitterContext const&)	
.text:000000710018DFC0 ; nn::audio::server::InfoUpdater::UpdatePerformanceBuffer(void *,ulong,nn::audio::server::PerformanceManager *)	
.text:000000710018DCE4 ; nn::audio::server::InfoUpdater::UpdateSinks(nn::audio::server::SinkContext &,nn::audio::server::MemoryPoolInfo *,int)	
.text:000000710018E178 ; nn::audio::server::InfoUpdater::UpdateSplitterInfo(nn::audio::server::SplitterContext &)	
.text:000000710018D2BC ; nn::audio::server::InfoUpdater::UpdateVoiceChannelResources(nn::audio::server::VoiceContext &)	
.text:000000710018D374 ; nn::audio::server::InfoUpdater::UpdateVoices(nn::audio::server::VoiceContext &,nn::audio::server::MemoryPoolInfo *,int)	`

`nn::audio::server::InfoUpdater::InfoUpdater` throws everything into `nn::audio::common::UpdateDataHeader::Initialize`

Comment (mailwl) [reply src/core/hle/service/audio/audren_u.cpp]: Ah, ok

**Pull request merged**